language-universal/src/main/java/io/github/rosemoe/sora/langs/desc/CppDescription.java

```diff
@@ -40,7 +40,8 @@
                 "return", "short", "signed", "sizeof", "static", "static_cast",
                 "struct", "switch", "template", "this", "throw", "true", "try",
                 "typedef", "typeid", "typename", "unsigned", "union",
-                "using", "virtual", "void", "volatile", "wchar_t", "while",
+                "using", "virtual", "void", "volatile", "wchar_t", "while", "define","include",
+          "_Bool", "char16_t","char32_t","cl","cin","do"
```

Comment:
Note that "do" is already defined in line 35, also "cin" is not a keyword.

Reply:
install acode and type cin